db/migrate/20250108221853_state_file_archived_intake_requests.rb

```diff
@@ -0,0 +1,14 @@
+class StateFileArchivedIntakeRequests < ActiveRecord::Migration[7.1]
+  def change
+    create_table :state_file_archived_intake_requests do |t|
+      t.belongs_to :state_file_archived_intakes, foreign_key: true
+      t.string 'ip_address'
+      t.string 'email_address'
+      t.jsonb 'details', default: '{}'
+      t.timestamps
+    end
+
+    remove_foreign_key :state_file_archived_intake_access_logs, :state_file_archived_intakes
+    safety_assured { remove_column :state_file_archived_intake_access_logs, :state_file_archived_intakes_id }
```

Comment:
do these lines do different things? they look redundant to me but i'm by no means an expert on this kind of thing